CRM/Contact/BAO/Query.php

```diff
@@ -6688,4 +6682,22 @@ protected function addModeToStats(&$summary, $from, $where) {
     $summary['total']['mode'] = CRM_Contribute_BAO_Contribution::computeStats('mode', $modeSQL);
   }
 
+  /**
+   * Add the median to the stats.
+   *
+   * Note that is can be a very slow query - taking many many minutes and even on a small
+   * data set it's likely to take longer than all the other queries combined by a significant
+   * multiple
+   *
+   * see https://lab.civicrm.org/dev/core/issues/720
+   *
+   * @param array $summary
+   * @param string $where
+   * @param string $from
+   */
+  protected function addMedianToStats(&$summary, $where, $from) {
+    $medianSQL = "{$from} {$where} ND civicrm_contribution.contribution_status_id = 1 ";
```

Comment:
`ND`?
Copy-paste error?
I'm interested to see if the tests catch it. If so then adding `merge-on-pass` is a no-brainer.

Reply:
opps - I re-tweaked that commit when pulling it out from the borg....  yeah tests will fail